src/Components/Links.js

```diff
@@ -12,6 +12,7 @@ function Links({ links }) {
     github: '#171515',
     instagram: '#E4405F',
     linkedin: '#0077b5',
+    codewars: '#A52A2A',
```

Comment:
Can you please add the screenshot or gif of the color or icon so it will be easy to know how it looks

Reply:
<img width="289" alt="image" src="https://user-images.githubusercontent.com/65719940/135564639-0f46259d-17bd-4441-9b07-99cf42e4715e.png">

Reply:
Here is the codewars icon

![](https://user-images.githubusercontent.com/70807500/135109170-6d7559f1-8624-436c-8364-259b19e4e5c7.png)

Reply:
codewars icon does exist in PrimeReact icon set https://www.primefaces.org/primereact/showcase/#/icons

We are going to change the icon set to include more